src/Compilers/Server/VBCSCompilerTests/VBCSCompilerServerTests.cs

```diff
@@ -14,6 +14,20 @@ namespace Microsoft.CodeAnalysis.CompilerServer.UnitTests
 {
     public class VBCSCompilerServerTests
     {
+        public class StartupTests : VBCSCompilerServerTests
+        {
+            [Fact]
+            [WorkItem(217709, "https://devdiv.visualstudio.com/DevDiv/_workitems/edit/217709")]
+            public void ShadowCopyAnalyzerAssemblyLoaderMissingDirectory()
+            {
+                var baseDirectory = Path.Combine(Path.GetTempPath(), TestBase.GetUniqueName());
+                var loader = new ShadowCopyAnalyzerAssemblyLoader(baseDirectory);
+                var task = loader.DeleteLeftoverDirectoriesTask;
+                task.Wait();
```

Comment:
I'd overlooked the option of making the test `async`. That's a better approach. Thanks.